game/scripts/vscripts/heroes/structures/healer_taste_of_armor.lua

```diff
@@ -38,4 +39,21 @@ function modifier_healer_taste_of_armor_effect:DeclareFunctions()
 end
 function modifier_healer_taste_of_armor_effect:GetModifierPhysicalArmorBonus()
 	return self:GetAbility():GetSpecialValueFor("aura_bonus_armor")
-end
\ No newline at end of file
+end
+modifier_healer_taste_of_armor_bottle_filling_delay = class({
```

Comment:
Wrong order, it looks like modifier_healer_taste_of_armor_effect:OnCreated() method is related to this modifier.